pkg/minikube/cruntime/cruntime.go

```diff
@@ -47,6 +46,11 @@ func (cs ContainerState) String() string {
 	return [...]string{"all", "running", "paused"}[cs]
 }
 
+// ValidOptions lists the supported container runtimes
+func ValidOptions() []string {
```

Comment:
plz use this in the start_flags.go so the help text uses this list to show to people what are the valid options.